docs/SECURITY.md

```diff
@@ -0,0 +1,81 @@
+
+We have made an effort to provide users of EVE with a system that is both practically secure and can be deployed in a zero-touch fashion. In order to achieve our goals we put forward a few guiding principles (some of them borrowed from Chromium's approach to securing mobile devices):
+
+* Make Edge Nodes secure-by-default
+* Defense in depth (based on the hypervisor boundary protection guarantees)
+* Relying on state-of-the-art hardware elements (such as TPM and TEE) for providing robust root-of-trust
+* Remote attestation and measured boot
+* Robust trust model between EVE and its controller
+* X509-based identity assigned to all the actors in the system
+* Workloads which are immutable
+* Secure overlay network
+
+In the rest of this document, we explain these principles and discuss some expected use cases for Edge Node devices running EVE.  We then give a high-level overview of the threat model against which we will endeavor to protect our users, while still enabling them to make full use of their cloud-based controller orchestration service.
+
+## EVE deployment model
+
+Reasoning about security and trustworthiness of any system is only meaningful if one understands how the systems is going to be deployed and used. EVE main deployment target are Edge Nodes. Typically, Edge Nodes are small form factor, ruggedized PC-like systems that are deployed in physically insecure environments with ad-hoc networking without any firewall-like guarantees. This type of environment is very similar to the one assumed by mobile computing systems (laptops, tablets and smart phones) and it is not a coincidence that EVE shares a lot of security principles with operating systems developed for mobile computing (e.g. Chromium OS and Android). For example, EVE's approach to thwarting physical attacks and protecting the integrity of the software stack running on the Edge Node 
+
+There is, however, one crucial difference between mobile computing environment and Edge and that is the notion of who controls the system. With mobile there's always an human being end user who has direct physical access to the hardware running mobile computing operating system, in EVE's case that entity is a remote, cloud-based controller. Any security related event that originates on EVE's side (e.g. measurements coming from boot sequence) has to be reliably passed and acknowledged by the controller and any action that is triggered on the Edge Node by EVE (e.g. storage key rotation) has to be scheduled by the controller.
```

Comment:
Question: "one crucial difference between mobile computing..." - this is the first mobile computing is raised?

Reply:
Not sure I understand your concern here. Could you elaborate?

Reply:
When I first looked at  this, something was missing in the previous paragraph about mobile computing. Might be related to the prior updated push. Looks fine now.